### PR TITLE
ODPS-4 Upload message UI

### PR DESCRIPTION
**Pull request by fruitbat2000**
Adds the UI component for file uploads

https://www.loom.com/share/1b38c58c353d4c26b1d892ae814c67c3 - a little out of date, but gives you the idea!

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ mix.js('resources/assets/js/app.js', './public/js')
     .js('resources/assets/js/opendialog-bot.js', './public/js')
     .js('resources/assets/js/opendialog-bot-full.js', './public/js')
     .sass('resources/assets/sass/main.scss', './public/css')
+    .sass('resources/assets/sass/client-custom.scss', './public/css')
```
Comment (patshone-gsl) [inline webpack.mix.js:20]: It looks like the brand styles are now included with main.scss so I don't quite understand why we're including the client-custom.scss here

**Review by patshone-gsl — APPROVED** [review]
Hey Vicki, this all looks fine - I don't quite understand the change with including the custom-client.scss but otherwise I think this is good to go

File: webpack.mix.js (hunk 1 above)
Comment (fruitbat2000) [reply webpack.mix.js:20]: You're absolutely right. The brand styles shouldn't have been included in the main css, I've removed them now

**Review by Lintume — APPROVED** [review]
works as expected, great!

**Pull request merged**